diff --git i/formats/MrssFormat.php w/formats/MrssFormat.php
index 4066ec73..131c8482 100644
--- i/formats/MrssFormat.php
+++ w/formats/MrssFormat.php
@@ -110,6 +110,7 @@ class MrssFormat extends FormatAbstract
 
         foreach ($this->getItems() as $item) {
             $itemArray = $item->toArray();
+            $itemAuthor = $item->getAuthor();
             $itemTimestamp = $item->getTimestamp();
             $itemTitle = $item->getTitle();
             $itemUri = $item->getURI();
@@ -137,6 +138,12 @@ class MrssFormat extends FormatAbstract
                 $entryTitle->appendChild($document->createTextNode($itemTitle));
             }
 
+            if (!empty($itemAuthor)) {
+                $entryAuthor = $document->createElement('author');
+                $entry->appendChild($entryAuthor);
+                $entryAuthor->appendChild($document->createTextNode($itemAuthor));
+            }
+
             if (isset($itemArray['itunes'])) {
                 $feed->setAttributeNS('http://www.w3.org/2000/xmlns/', 'xmlns:itunes', self::ITUNES_NS);
                 foreach ($itemArray['itunes'] as $itunesKey => $itunesValue) {
