diff --git a/formats/MrssFormat.php b/formats/MrssFormat.php
index aaa1d0c..79e0dbd 100644
--- a/formats/MrssFormat.php
+++ b/formats/MrssFormat.php
@@ -119,6 +119,7 @@ class MrssFormat extends FormatAbstract
             $itemTimestamp = $item->getTimestamp();
             $itemTitle = $item->getTitle();
             $itemUri = $item->getURI();
+            $itemComments = $item->getComments();
             $itemContent = $item->getContent() ?? '';
             $itemUid = $item->getUid();
             $isPermaLink = 'false';
@@ -166,6 +167,12 @@ class MrssFormat extends FormatAbstract
                 $entryLink->appendChild($document->createTextNode($itemUri));
             }
 
+            if (!empty($itemComments)) {
+                $entryComments = $document->createElement('comments');
+                $entry->appendChild($entryComments);
+                $entryComments->appendChild($document->createTextNode($itemComments));
+            }
+
             $entryGuid = $document->createElement('guid');
             $entryGuid->setAttribute('isPermaLink', $isPermaLink);
             $entry->appendChild($entryGuid);
diff --git a/lib/FeedItem.php b/lib/FeedItem.php
index 8c9a60b..b408d24 100644
--- a/lib/FeedItem.php
+++ b/lib/FeedItem.php
@@ -3,6 +3,7 @@
 class FeedItem
 {
     protected ?string $uri = null;
+    protected ?string $comments = null;
     protected ?string $title = null;
     protected ?int $timestamp = null;
     protected ?string $author = null;
@@ -34,6 +35,9 @@ class FeedItem
             case 'uri':
                 $this->setURI($value);
                 break;
+            case 'comments':
+                $this->setComments($value);
+                break;
             case 'title':
                 $this->setTitle($value);
                 break;
@@ -65,6 +69,8 @@ class FeedItem
         switch ($name) {
             case 'uri':
                 return $this->getURI();
+            case 'comments':
+                return $this->getComments();
             case 'title':
                 return $this->getTitle();
             case 'timestamp':
@@ -92,10 +98,8 @@ class FeedItem
         return $this->uri;
     }
 
-    public function setURI($uri)
+    private function validateURI($uri): ?string
     {
-        $this->uri = null; // Clear previous data
-
         if ($uri instanceof simple_html_dom_node) {
             if ($uri->hasAttribute('href')) { // Anchor
                 $uri = $uri->href;
@@ -107,15 +111,30 @@ class FeedItem
         }
         if (!is_string($uri)) {
             $this->logger->debug(sprintf('Expected $uri to be string but got %s', gettype($uri)));
-            return;
+            return null;
         }
         $uri = trim($uri);
         // Intentionally doing a weak url validation here because FILTER_VALIDATE_URL is too strict
         if (!preg_match('#^https?://#i', $uri)) {
             $this->logger->debug(sprintf('Not a valid url: "%s"', $uri));
-            return;
+            return null;
         }
-        $this->uri = $uri;
+        return $uri;
+    }
+
+    public function setURI($uri)
+    {
+        $this->uri = $this->validateURI($uri);
+    }
+
+    public function getComments(): ?string
+    {
+        return $this->comments;
+    }
+
+    public function setComments($uri)
+    {
+        $this->comments = $this->validateURI($uri);
     }
 
     public function getTitle(): ?string
@@ -283,6 +302,7 @@ class FeedItem
         return array_merge(
             [
                 'uri' => $this->uri,
+                'comments' => $this->comments,
                 'title' => $this->title,
                 'timestamp' => $this->timestamp,
                 'author' => $this->author,
