diff --git i/formats/MrssFormat.php w/formats/MrssFormat.php
index 4066ec73..f6321113 100644
--- i/formats/MrssFormat.php
+++ w/formats/MrssFormat.php
@@ -113,6 +113,7 @@ class MrssFormat extends FormatAbstract
             $itemTimestamp = $item->getTimestamp();
             $itemTitle = $item->getTitle();
             $itemUri = $item->getURI();
+            $itemComments = $item->getComments();
             $itemContent = $item->getContent() ?? '';
             $itemUid = $item->getUid();
             $isPermaLink = 'false';
@@ -160,6 +161,12 @@ class MrssFormat extends FormatAbstract
                 $entryLink->appendChild($document->createTextNode($itemUri));
             }
 
+            if (!empty($itemComments)) {
+                $entryComments = $document->createElement('comments');
+                $entry->appendChild($entryComments);
+                $entryComments->appendChild($document->createTextNode($itemComments));
+            }
+
             $entryGuid = $document->createElement('guid');
             $entryGuid->setAttribute('isPermaLink', $isPermaLink);
             $entry->appendChild($entryGuid);
diff --git i/lib/FeedItem.php w/lib/FeedItem.php
index 5fb9240e..82755621 100644
--- i/lib/FeedItem.php
+++ w/lib/FeedItem.php
@@ -3,6 +3,7 @@
 class FeedItem
 {
     protected ?string $uri = null;
+    protected ?string $comments = null;
     protected ?string $title = null;
     protected ?int $timestamp = null;
     protected ?string $author = null;
@@ -31,6 +32,9 @@ class FeedItem
             case 'uri':
                 $this->setURI($value);
                 break;
+            case 'comments':
+                $this->setComments($value);
+                break;
             case 'title':
                 $this->setTitle($value);
                 break;
@@ -64,6 +68,8 @@ class FeedItem
                 return $this->getURI();
             case 'title':
                 return $this->getTitle();
+            case 'comments':
+                return $this->getComments();
             case 'timestamp':
                 return $this->getTimestamp();
             case 'author':
@@ -89,10 +95,8 @@ class FeedItem
         return $this->uri;
     }
 
-    public function setURI($uri)
+    private function validateURI($uri): ?string
     {
-        $this->uri = null;
-
         if ($uri instanceof simple_html_dom_node) {
             if ($uri->hasAttribute('href')) { // Anchor
                 $uri = $uri->href;
@@ -101,16 +105,31 @@ class FeedItem
             }
         }
         if (!is_string($uri)) {
-            return;
+            return null;
         }
         $uri = trim($uri);
         // Intentionally doing a weak url validation here because FILTER_VALIDATE_URL is too strict
         if (!preg_match('#^https?://#i', $uri)) {
-            return;
+            return url;
         }
-        $this->uri = $uri;
+        return $uri;
     }
 
+    public function setURI($uri)
+    {
+        $this->uri = $this->validateURI($uri);
+    }
+
+    public function getComments(): ?string
+    {
+        return $this->comments;
+    }
+
+    public function setComments($uri)
+    {
+        $this->comments = $this->validateURI($uri);
+     }
+
     public function getTitle(): ?string
     {
         return $this->title;
@@ -253,6 +272,7 @@ class FeedItem
         return array_merge(
             [
                 'uri' => $this->uri,
+                'comments' => $this->comments,
                 'title' => $this->title,
                 'timestamp' => $this->timestamp,
                 'author' => $this->author,
