diff --git a/bridges/PriviblurBridge.php b/bridges/PriviblurBridge.php
index 198c38bc6c3..6b442e75034 100644
--- a/bridges/PriviblurBridge.php
+++ b/bridges/PriviblurBridge.php
@@ -17,6 +17,7 @@ class PriviblurBridge extends BridgeAbstract
     ];
 
     private $title;
+    private $favicon = 'https://www.tumblr.com/favicon.ico';
 
     public function collectData()
     {
@@ -25,6 +26,11 @@ public function collectData()
         $html = defaultLinkTo($html, $url);
         $this->title = $html->find('head title', 0)->innertext;
 
+        if ($html->find('#blog-header img.avatar', 0)) {
+            $icon = $html->find('#blog-header img.avatar', 0)->src;
+            $this->favicon = str_replace('pnj', 'png', $icon);
+        }
+
         $elements = $html->find('.post');
         foreach ($elements as $element) {
             $item = [];
@@ -64,6 +70,11 @@ public function getName()
 
     public function getURI()
     {
-        return $this->getInput('url') ? $this->getInput('url') : parent::getURI();
+        return $this->getInput('url') ?? parent::getURI();
+    }
+
+    public function getIcon()
+    {
+        return $this->favicon;
     }
 }
