diff --git i/Dockerfile w/Dockerfile
index fb783344..fb7bc4cd 100644
--- i/Dockerfile
+++ w/Dockerfile
@@ -27,6 +27,8 @@ RUN set -xe && \
       curl \
       # for patching libcurl-impersonate
       patchelf \
+      # to install readability
+      composer \
       && \
     # install curl-impersonate library
     curlimpersonate_version=1.0.0rc2 && \
@@ -69,6 +71,7 @@ COPY ./config/php-fpm.conf /etc/php/8.2/fpm/pool.d/rss-bridge.conf
 COPY ./config/php.ini /etc/php/8.2/fpm/conf.d/90-rss-bridge.ini
 
 COPY --chown=www-data:www-data ./ /app/
+RUN composer require "fivefilters/readability.php:*" -d /app
 
 EXPOSE 80
 
