diff --git i/config/nginx.conf w/config/nginx.conf
index c65f8e00..6e9013e2 100644
--- i/config/nginx.conf
+++ w/config/nginx.conf
@@ -16,4 +16,8 @@ server {
         fastcgi_read_timeout 45s;
         fastcgi_pass unix:/var/run/php/php8.2-fpm.sock;
     }
+
+    location / {
+        try_files $uri $uri/ /index.php;
+    }
 }
diff --git i/lib/RssBridge.php w/lib/RssBridge.php
index c7b132d6..742649bc 100644
--- i/lib/RssBridge.php
+++ w/lib/RssBridge.php
@@ -10,8 +10,106 @@ final class RssBridge
         $this->container = $container;
     }
 
+    private static function rewrite(Request $request): Request
+    {
+        $path = $request->server('REQUEST_URI');
+        if (str_starts_with($path, '/tags')) {
+            $path = '/ao3' . $path;
+        }
+        if (str_starts_with($path, '/ao3')) {
+            $args = [
+                'context' => 'List',
+                'range' => 'all',
+                'url' => 'https://archiveofourown.org' . substr($path, 4),
+            ];
+            $request = Request::fromCli($args);
+        }
+        if (str_starts_with($path, '/manga') || str_starts_with($path, '/title')) {
+            $args = [
+                'series' => substr($path, 7),
+            ];
+            $path = '/cubariproxy';
+            $request = Request::fromCli($args);
+        }
+        if (str_starts_with($path, '/tagged/')) {
+            $path = '/blog' . $path . '?sort=recent';
+        }
+        if (str_starts_with($path, '/blog/')) {
+            $loc = substr($path, 6);
+            $args = [
+                'url' => 'https://priviblur.fly.dev/' . $loc
+            ];
+            $path = '/priviblur';
+            $request = Request::fromCli($args);
+        }
+        if (str_starts_with($path, '/profile/')) {
+            $args = [
+                'context' => 'Author',
+                'uid' => substr($path, 8),
+            ];
+            $path = '/scribblehub';
+            $request = Request::fromCli($args);
+        }
+        if (str_starts_with($path, '/channel/')) {
+            $args = [
+                'embed' => 'on',
+                'embedurl' => 'on',
+                'channel' => substr($path, 9),
+            ];
+            $path = '/youtubefeedexpander';
+            $request = Request::fromCli($args);
+        }
+        if (str_starts_with($path, '/patreon/')) {
+            $args = [
+                'user' => substr($path, 9),
+            ];
+            $path = '/kemono';
+            $request = Request::fromCli($args);
+        }
+        if (str_starts_with($path, '/games/')) {
+            $args = [
+                'updates' => on,
+                'gid' => substr($path, 7),
+            ];
+            $path = '/gamebanana';
+            $request = Request::fromCli($args);
+        }
+        if (str_starts_with($path, '/@')) {
+            $args = [
+                'profile' => substr($path, 2),
+            ];
+            $path = '/substackprofile';
+            $request = Request::fromCli($args);
+        }
+        if (str_starts_with($path, '/reduce/')) {
+            $args = [
+                'url' => substr($path, 8),
+                'percentage' => '50',
+            ];
+            $path = '/feedreducer';
+            $request = Request::fromCli($args);
+        }
+        if (str_starts_with($path, '/bsky/')) {
+            $args = [
+                'user_id' => substr($path, 6) . '.bsky.social',
+            ];
+            $path = '/bluesky';
+            $request = Request::fromCli($args);
+        }
+        if (strlen($path) > 1 && $path[1] != '?') {
+            $args = $request->toArray() + [
+                'action' => 'display',
+                'bridge' => explode('/', $path)[1],
+                'format' => 'Mrss',
+            ];
+            $request = Request::fromCli($args);
+        }
+        return $request;
+    }
+
     public function main(Request $request): Response
     {
+        $request = $this::rewrite($request);
         $action = $request->get('action', 'Frontpage');
         $actionName = strtolower($action) . 'Action';
         $actionName = implode(array_map('ucfirst', explode('-', $actionName)));
