diff --git i/config/nginx.conf w/config/nginx.conf
index c65f8e00..6e9013e2 100644
--- i/config/nginx.conf
+++ w/config/nginx.conf
@@ -16,4 +16,8 @@ server {
         fastcgi_read_timeout 45s;
         fastcgi_pass unix:/var/run/php/php8.2-fpm.sock;
     }
+
+    location / {
+        try_files $uri $uri/ /index.php;
+    }
 }
diff --git i/lib/RssBridge.php w/lib/RssBridge.php
index d16f1d89..801b97f5 100644
--- i/lib/RssBridge.php
+++ w/lib/RssBridge.php
@@ -10,8 +10,95 @@ final class RssBridge
         $this->container = $container;
     }
 
+    private static function rewrite(Request $request): Request
+    {
+        $rewriteRules = [
+            '/tags/' => [
+                'path' => fn($p) => '/ao3' . $p . '/works?work_search[sort_column]=created_at',
+            ],
+
+            '/@' => [
+                'bridge' => 'substackprofile',
+                'handler' => fn($p) => ['profile' => substr($p, 2)]
+            ],
+            '/ao3/' => [
+                'bridge' => 'ao3',
+                'handler' => fn($p) => [
+                    'context' => 'List',
+                    'range' => 'all',
+                    'url' => 'https://archiveofourown.org' . substr($p, 4)
+                ]
+            ],
+            '/bsky/' => [
+                'bridge' => 'bluesky',
+                'handler' => fn($p) => ['user_id' => substr($p, 6) . '.bsky.social']
+            ],
+            '/channel/' => [
+                'bridge' => 'youtubefeedexpander',
+                'handler' => fn($p) => [
+                    'embed' => 'on',
+                    'embedurl' => 'on',
+                    'channel' => substr($p, 9)
+                ]
+            ],
+            '/game/' => [
+                'bridge' => 'ggdeals',
+                'handler' => fn($p) => ['slug' => substr($p, 6)]
+            ],
+            '/patreon/' => [
+                'bridge' => 'kemono',
+                'handler' => fn($p) => ['user' => substr($p, 9)]
+            ],
+            '/profile/' => [
+                'bridge' => 'scribblehub',
+                'handler' => fn($p) => [
+                    'context' => 'Author',
+                    'uid' => substr($p, 9)
+                ]
+            ],
+            '/reduce/' => [
+                'bridge' => 'feedreducer',
+                'handler' => fn($p) => [
+                    'url' => substr($p, 8),
+                    'percentage' => '50'
+                ]
+            ],
+            '/title/' => [
+                'bridge' => 'cubariproxy',
+                'handler' => fn($p) => ['series' => substr($p, 7)]
+            ],
+        ];
+
+        $path = $request->server('REQUEST_URI');
+        foreach ($rewriteRules as $prefix => $rule) {
+            if (str_starts_with($path, $prefix)) {
+                if (isset($rule['path'])) {
+                    $path = $rule['path']($path);
+                    continue;
+                }
+
+                $args = $rule['handler']($path);
+                $path = '/' . $rule['bridge'];
+                $request = Request::fromCli($args);
+                break;
+            }
+        }
+
+        if (strlen($path) > 1 && $path[1] != '?') {
+            $args = $request->toArray() + [
+                'action' => 'display',
+                'bridge' => preg_split('/[\/\?]/', $path)[1],
+                'format' => 'Mrss',
+            ];
+            $request = Request::fromCli($args);
+        }
+
+        return $request;
+    }
+
     public function main(Request $request): Response
     {
+        $request = $this::rewrite($request);
         $action = $request->get('action', 'Frontpage');
         $actionName = strtolower($action) . 'Action';
         $actionName = implode(array_map('ucfirst', explode('-', $actionName)));
